### PR TITLE
First revision for the updater

### PR DESCRIPTION
**Pull request by x0rz3q**

### CODE REVIEW

File: b0tweb/Commands/UpdateCommand.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            };
+
+            Process process = new Process()
+            {
+                StartInfo = info
+            };
+
+            process.Start();
+
+            Environment.Exit(0);
```
Comment (JurisMajors) [inline b0tweb/Commands/UpdateCommand.cs]: Won't this kill all child processes? including the new b0tweb processl.

Comment (x0rz3q) [reply b0tweb/Commands/UpdateCommand.cs]: It does not, it only kills the parent and lets the other process run free.

File: b0tweb/FileHelper.cs [hunk 2]
```diff
@@ -43,5 +43,15 @@ public static void DeleteFile(string path)
 
             File.Delete(path);
         }
+
+        public static string GetBinaryPath()
```
Comment (x0rz3q) [inline b0tweb/FileHelper.cs]: Docs

File: b0tweb/FileHelper.cs [hunk 3]
```diff
@@ -43,5 +43,15 @@ public static void DeleteFile(string path)
...
+        {
+            return FileHelper.GetBasePath() + @"\b0tweb.exe";
+        }
+
+        public static string GetBinaryOldPath()
```
Comment (x0rz3q) [inline b0tweb/FileHelper.cs]: Docs

**Pull request merged**